packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -179,6 +198,7 @@ const AuthDefault = ({
               button: shouldLogin ? 'Sign up' : 'Log in',
             }}
             onClick={() => {
+              focusFirstSocialLink();
```

Comment:
By clicking this element the component will refresh and we want to move the focus on top just to make the navigation easier